### PR TITLE
/

### PR DESCRIPTION
**Pull request by sueszli**
Added possibility to export other files in addition to .pdf files.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,47 @@
+beautifulsoup4==4.9.1 \
```
Comment (sueszli) [inline requirements.txt:1]: leaving poetry is a great idea

File: notionsnapshot/__main__.py [hunk 2]
```diff
@@ -359,12 +394,12 @@ def _download_stylesheets(soup: BeautifulSoup) -> None:
 
     @trace()
     @staticmethod
-    def _download_pdfs(soup: BeautifulSoup) -> None:
```
Comment (sueszli) [inline notionsnapshot/__main__.py:368]: supporting generic support for all kinds of files is a great idea - but i think we might need to reconfigure the chrome driver ... not sure still have to check

File: requirements.txt (hunk 1 above)
Comment (sueszli) [reply requirements.txt:1]: as we currently already have to use both poetry + pip

File: README.md [hunk 3]
```diff
@@ -1,18 +1,23 @@
+# Why this fork?
```
Comment (sueszli) [inline README.md:1]: was the idea contributing or starting from scratch?

you could add your name at line 114

File: notionsnapshot/__main__.py [hunk 4]
```diff
@@ -230,6 +245,26 @@ def is_page_loaded(d: webdriver.Chrome) -> bool:
         Scraper.driver.execute_script("__console.environment.ThemeStore.setState({ mode: '" + mode + "' })")
         LOG.info(f"set theme to {mode}-mode")
 
+    @staticmethod
+    def _create_dictionary(url: str, soup: BeautifulSoup) -> dict[str, object]:
```
Comment (sueszli) [inline notionsnapshot/__main__.py]: i wouldn't add this feature as i think it serves a really really niche and specific need

**Review by sueszli — COMMENTED** [review]
thank you so much for checking out my project and forking it :)
i added a couple of comments here and there.

File: notionsnapshot/__main__.py (hunk 4 above)
Comment (divyaeM) [reply notionsnapshot/__main__.py]: Yes, this feature is very specific to my usecase.

Apparently, Notion links works differently for Embeded link vs direct link. This results in exporting of duplicate pages. So this dictionary helped me remove the duplicates.

File: README.md (hunk 3 above)
Comment (divyaeM) [reply README.md:1]: The idea was to fork for a specific use case. However, if you want to take some learnings, happy to contribute.

**Pull request closed without merging**